Plugin/CsrfValidatorSkip.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Twispay\Payments\Plugin;
+
+/**
+ * Twispay payment method  CSRF validator
+ *
+ * @category    Twispay\Payments\Plugin
+ * @package     Twispay_Payments
+ * @author      Twispay
+ */
+class CsrfValidatorSkip {

```

Comment:
Why we need this?

Reply:
Starting from Magento 2.3, all outside POST requests must go through a CSRF check.
There are two approaches for this:
  1. Implement the "CsrfAwareActionInterface" interface in the controller (in our case is "Twispay\Payments\Controller\Payment\BackUrl") satisfying the external POST request
      - This would meet the requirement but it also will make the plugin backwards incompatible as the "CsrfAwareActionInterface" was introduced in Magento 2.3
  2. Implement a plugin type object (in out case is "Twispay\Payments\Plugin\CsrfValidatorSkip") that would alter the flow of the external POST request like following:
      - If the request is for the "twispay" plugin we will skip the CSRF check in order to make keep the plugin backwards compatible;
      - If the request is NOT for the "twispay" plugin we will proceed with the normal flow so that nothing else besides the "twispay" plugin is changed.